system/default_ad_api_helpers/src/routing_adaptor.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "routing_adaptor.hpp"
+
+#include <memory>
+
+namespace default_ad_api_helpers
+{
+
+RoutingAdaptor::RoutingAdaptor() : Node("routing_adaptor")
+{
+  RCLCPP_INFO_STREAM(get_logger(), "RoutingAdaptor");
+
+  sub_goal_ = create_subscription<PoseStamped>(
+    "input/goal", 5, std::bind(&RoutingAdaptor::OnGoal, this, std::placeholders::_1));
+  sub_waypoints_ = create_subscription<PoseStamped>(
+    "input/checkpoint", 5, std::bind(&RoutingAdaptor::OnWaypoint, this, std::placeholders::_1));
```

Comment:
[IMO] I feel using `waypoint` here is natural.

```suggestion
  sub_waypoint_ = create_subscription<PoseStamped>(
    "input/waypoint", 5, std::bind(&RoutingAdaptor::OnWaypoint, this, std::placeholders::_1));
```

Reply:
Done https://github.com/autowarefoundation/autoware.universe/pull/1494/commits/3391860130b61e46465e9a3db50e794c8c747f80.